<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentWindown</class>
 <widget class="QWidget" name="ContentWindown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="user_image">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>chenyiyun</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>搜索：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
